vic-class08-assignment/test/persistance_test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const fs = require('fs');
+const chai = require('chai');
+const chaiHTTP = require('chai-http');
+const expect = chai.expect;
+chai.use(chaiHTTP);
+const request = chai.request;
+
+require('../server');
+
+describe('HTTP test', () => {
+  let arrayLength;
+  before((done) => {
+    fs.readdir(__dirname + '/../data', (err, data) => {
+      if (err) throw err;
+      arrayLength = data.length;
+      done();
+    });
+  });
+  it('should return a list of files', (done) => {
+    request('localhost:3000')
+    .get('/notes')
+    .end((err, res) => {
+      console.log(res);
```

Comment:
Probably don't need to log the entire response out in your test.  I'm running our of space in my terminal. :-)